src/zeit/content/article/edit/browser/edit.py

```diff
@@ -128,16 +128,27 @@ def writeable(self):
     def sortable(self):
         return zope.security.canAccess(self.context, 'updateOrder')
 
+    def values(self):
+        return self.context.values()
+
     @property
     def body_css_class(self):
         css_class = ['editable-area']
         if self.sortable:
             css_class.append('action-block-sorter')
         return ' '.join(css_class)
 
+
+class EditableHeaderArea(object):
+
     @property
-    def values(self):
-        return self.context.values()
+    def body_css_class(self):
```

Comment:
Man könnte statt ner CSS-Klasse den View auch insgesamt _nichts_ ausgeben lassen (indem man `__call__` überschreibt und ggf. `''` zurückgibt). Das machen wir an anderen Stellen auch, um Dinge auszublenden, und das hätte noch die Mikrooptimierung drin, dass man den Header in dem Fall gar nicht parsen/rendern müsste. (Und da das UI eh ne Reload dieses HTML-Teils macht, spart man mit der CSS-Klasse auch später leider nichts.)